PrestaShop/PrestaShop
# Unify page names, grid names, form names

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | See below
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Browse all backoffice and check page titles.
| Fixed ticket?     | 
| Related PRs       | 
| Sponsor company   | 

### How to test
- Enable all feature flags.
- Browse backoffice and see nothing is broken.

### Changes
- Fixed HTML syntax of some card headers.
- Unified grid names - they should be in plural.
- Unified editing forms card names - their names should be singular.
- Unified a place where `enableSidebar` is defined - in the controllers.
- Unified a place where `layoutTitle` is defined - in the controllers. Exception is the more complicated one on order page.
- Added and unified missing page names (`layoutTitle`) everywhere.
  - **Listings:** `Categories`, `Brands`, `Tax rules`
  - **Detail:** `Brand Studio design`, `Result of MySQLRequest`, `Customer Daniel Hlaváček`, `Cart #1548`
  - **New:** `New category`, `New brand`, `New tax rule`
  - **Editation:** `Editing category Women`, `Editing brand Studio design`, `Editing tax rule CZ 21 %`
  - _I followed an example from Customers section, where it's done somehow correctly. For titles with IDs I kept the original logic with a hashtag, used throughout the backoffice._
- Added specific layoutTitles to every controller so they don't use a fallback from left menu.
_____________________________

### Layout title changes

If the change is around `'layoutTitle' =>` and the filename ends with `...Controller.php`, it!s a page name. On almost all places except Customer and Category sections, they were not done or not unified at least.

![layoutTitle](https://user-images.githubusercontent.com/6097524/219432034-03bdb0c5-829d-41bd-8133-c68f59a7e1f3.jpg)

-------------------------

### Form title changes
If the file with the change is `form.html.twig`, it is a header of an editation form. There were some form titles that were in plural, although they are concerned with a single entity.

![form title](https://user-images.githubusercontent.com/6097524/219432181-5a27f600-c88e-42b9-a939-a0eb53bd046a.jpg)

-------------------------

### Grid title changes
Everything that is wrapped around `protected function getName()` and the filename contains `Factory` is a grid. There were only a few fixes, mostly singular/plural form.

![grid](https://user-images.githubusercontent.com/6097524/219432845-9db3cef4-ce31-4d4e-a475-b50aee984260.jpg)
